src/OneExplorer/OneExplorer.ts

```diff
@@ -893,23 +888,23 @@ input_path=${modelName}.${extName}
 
   getChildren(element?: Node): vscode.ProviderResult<Node[]> {
     if (!element) {
-      element = this.getTree();
+      return this.getTree();
     }
 
-    return element ?.getChildren();
+    return element.getChildren();
   }
 
   /**
    * Get the root of the tree
    */
-  private getTree(): Node|undefined {
-    if (!this.workspaceRoot) {
+  private getTree(): Node[]|undefined {
+    if (!this._workspaceRoots || this._workspaceRoots.length === 0) {
       return undefined;
```

Comment:
Not directly related with this PR, but how about returning empty array instead of `undefined`? :)
I am not sure but `getChildren(element)` can return `undefined` if `getTree()` returns `undefined`...!
(But it seems to work well currently @.@)

Reply:
Yes, I understand the point of your suggestion. That's actually the exact strategy I used in the previous PR(introducing obtainWorkspaceRoots) to clarify the return and simplify the caller. Here, it works correctly in both ways and the caller function doesn't bother as it returns vscode.ProviderResult<Node[]>. I think it's optional, so I would like to proceed as is -  I wouldn't object to it if you create the PR, though :-D